test/docfx.Tests/BuildCommandTest.cs

```diff
@@ -169,11 +169,11 @@ public void TestGetConfigWithNoInputAndDocfxJsonExists()
                 new string[]
                 {
                     "",
-                    $"<h1 id=\"hello-test1\" sourcefile=\"{_inputFolder}/test1.md\" sourcelinenumber=\"7\">Hello Test1</h1>",
-                    $"<p sourcefile=\"{_inputFolder}/test1.md\" sourcelinenumber=\"8\">Test XRef: <a class=\"xref\" href=\"test2.html#xref2\">Hello World</a>",
+                    $"<h1 id=\"hello-test1\" sourcefile=\"{_inputFolder}/test1.md\" sourcestartlinenumber=\"7\" sourceendlinenumber=\"7\">Hello Test1</h1>",
+                    $"<p sourcefile=\"{_inputFolder}/test1.md\" sourcestartlinenumber=\"8\" sourceendlinenumber=\"10\">Test XRef: <a class=\"xref\" href=\"test2.html#xref2\">Hello World</a>",
```

Comment:
Got it. And the method is not in namespace system.windows.forms, but in HtmlAgilityPack.

---
In reply to: [76957651](https://github.com/dotnet/docfx/pull/654#discussion_r76957651) [](ancestors = 76957651,76949337)